lib/iris/tests/unit/coords/test_AuxCoord.py

```diff
@@ -0,0 +1,613 @@
+# (C) British Crown Copyright 2017, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Unit tests for the :class:`iris.coords.AuxCoord` class.
+
+Note: a lot of these methods are actually defined by the :class:`Coord` class,
+but can only be tested on concrete DimCoord/AuxCoord instances.
+In addition, the DimCoord class has little behaviour for some of these, as it
+cannot contain lazy points or bounds data.
+
+"""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+
+import numpy as np
+
+from iris.tests.unit.coords import (CoordTestMixin,
```

Comment:
I tried that out, but then I found you can't then run the test as a standalone script any more.
You get a message like ...
```
$ python lib/iris/tests/unit/coords/test_AuxCoord.py Test_points__setter
Traceback (most recent call last):
  File "lib/iris/tests/unit/coords/test_AuxCoord.py", line 34, in <module>
    from . import (CoordTestMixin, lazyness_string,
ValueError: Attempted relative import in non-package

$ 
```

So, probably best left as it is.